index.html

```diff
@@ -551,14 +569,16 @@ <h1>How can I make a button go full width?</h1>
         <div class="columns">
           <div class="column is-3 is-offset-4">
             <div class="is-3 title"><b>News</b></div>
-            <ul>
+            <!-- WARN: Role is useless, added for HW -->
+            <ul role="list">
```

Comment:
Yes, you are completely right, it is useless as `ul` is semantically a list. It's unnecessary to specify the role unless you're overriding a non-semantic use. So you can freely remove it